content/1_General/Cpp_Command.mdx

```diff
@@ -184,22 +180,86 @@ on your needs. To find more information, go to
     that will almost surely cause problems. Don't worry if you don't know what
     those terms mean.
 
-<!-- <Resources>
+### On Windows
+
+#### Simpler: [Mingw-w64](https://www.mingw-w64.org/) (Minimalist GNU for Windows)
+
+<Resources>
+	<Resource
+		source="Microsoft"
+		title="MinGW with VS Code"
+		url="https://code.visualstudio.com/docs/cpp/config-mingw"
+	>
+		Setting Up MinGW with VS Code
+	</Resource>
 	<Resource
-		source="Benq"
-		title="My Template"
-		url="https://github.com/bqi343/USACO/blob/master/Implementations/content/contest/template.cpp"
+		source="Jetbrains"
+		title="Configuring CLion on Windows"
+		url="https://www.jetbrains.com/help/clion/quick-tutorial-on-configuring-clion-on-windows.html#MinGW"
 	>
+		Setting up MinGW with CLion
 	</Resource>
+</Resources>
```

Comment:
Does mingw have support for `bits/stdc++.h`? I vaguely remember that it doesn't, but I could be wrong.

Potentially relevant links:

- https://codeforces.com/blog/entry/13981
- https://codeforces.com/blog/entry/73240

Reply:
Looks like these refer to MinGW (as opposed to Mingw-w64), though I have no idea whether there's a difference? https://github.com/cpinitiative/usaco-guide/commit/a7a14522db155a0aefcd79d48e68cdddcc0ebc12 seems to think there is.